models/balancer/ethereum/balancer_ethereum_schema.yml

```diff
@@ -1,13 +1,65 @@
 version: 2
 
 models:
+  - name: balancer_v2_ethereum_pools_fees
```

Comment:
have you tried to run on other networks? I'd be curious how much time it takes to run on Polygon.

Reply:
On Polygon, the query returns a timeout error after running for 30 minutes.

I will have to think about how to optimize this query, maybe by filter by `block_number`.

Reply:
I re-ran the query again for the Polygon network and it finished in ~25 minutes.